easybuild/easyconfigs/m/MCR/MCR-R2021b.eb

```diff
@@ -0,0 +1,20 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'MCR'
+version = 'R2021b'  # runtime version 9.10
```

Comment:
shouldn't the comment be changed to 9.11?

Reply:
fixed in 9b13324

Reply:
@boegel thank you.